app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,48 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+  padding-bottom: govuk-spacing(8);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(8);
+  }
+}
+
+.homepage-search {
+  @include govuk-media-query($until: "tablet") {
+    max-width: 420px;
+  }
+}
+
+.homepage-most-viewed-list {
+  list-style: none;
+  margin: 0;
+  padding: 0 0 govuk-spacing(7) 0;
+
+  @include govuk-media-query($from: "tablet") {
+    padding-bottom: 0;
+  }
+
+  & > li {
+    padding-bottom: govuk-spacing(4);
```

Comment:
This doesn't feel quite right using padding to dictate spacing between the list items. Could it be margin instead?

Reply:
Sure - swapped in https://github.com/alphagov/frontend/pull/3028/commits/05f881c70c9ff75a0a12080d09ffdd1ee4aa8ca2.